openedx/features/enterprise_support/admin/views.py

```diff
@@ -0,0 +1,87 @@
+from django.views.generic import View
+from django.contrib import admin, messages
+from django.shortcuts import render
+from django.urls import reverse
+from django.http import HttpResponseRedirect
+from django.utils.translation import ugettext as _
+
+from openedx.features.enterprise_support.admin.forms import CSVImportForm
+from student.models import CourseEnrollment, CourseEnrollmentAttribute
+
+from enterprise.admin.utils import parse_csv
+from enterprise.models import EnterpriseCourseEnrollment
+
+from django.core.exceptions import ValidationError
+
+
+class EnrollmentAttributeOverrideView(View):
+    """
+    Learner Enrollment Attribute Override View.
+    """
+    template = 'enterprise_support/admin/enrollment_attributes_override.html'
+
+    @staticmethod
+    def _get_admin_context(request):
+        admin_context = {'opts': EnterpriseCourseEnrollment._meta}
+        admin_context.update(admin.site.each_context(request))
+
+        return admin_context
+
+    def get(self, request):
+        """
+        Render Enrollment Attribute Override View.
+        """
+        context = {'csv_form': CSVImportForm()}
+        context.update(self._get_admin_context(request))
+
+        return render(request, self.template, context)
+
+    def post(self, request):
+        """
+        HTTP POST handler for Enrollment Attribute Override View
```

Comment:
Can you add more information about what this method does? e.g. creates enrollment attributes for courses etc.